src/store/useAppConfigStore.ts

```diff
@@ -41,7 +44,7 @@ const initialState: State = {
     config_version: 0,
     config_file: undefined,
     mode: 'Eco',
-    auto_mining: true,
```

Comment:
Was this the feature gate for `mine_on_app_start`?

Reply:
nope that'd be [`mine_on_app_start`](https://github.com/tari-project/universe/pull/1090/files#diff-cd1133f102d8444a5d366fbd1e7d5878e3908ba7f1d41ea778a6893426af8a63R48) `auto_mining` is the flag from aaages ago